### PR TITLE
Internal 126 lens failing

### PR DESCRIPTION
**Pull request by andrea-grapl**
### Which issue does this PR correspond to?
https://github.com/grapl-security/issue-tracker/issues/126

### What changes does this PR make to Grapl? Why?
Changed `graphql` endpoints in `server.js` to accommodate needs for local development + API gateway.

### How were these changes tested?
Locally + Manually.

### CODE REVIEW

File: src/js/graphql_endpoint/server.js [hunk 1]
```diff
@@ -56,10 +56,17 @@ const middleware = [cors(corsDelegate), validateJwt];
...
 app.options('*', cors(corsDelegate));
 
-app.use('/graphQlEndpoint/graphql', middleware, graphqlHTTP({
-    schema: schema,
-    graphiql: IS_LOCAL !== null
-}));
+if (IS_LOCAL) {
+    app.use('/graphQlEndpoint/graphql', middleware, graphqlHTTP({
+        schema: schema,
+        graphiql: IS_LOCAL !== null
```
Comment (colin-grapl) [inline src/js/graphql_endpoint/server.js:62]: This can just be `graphiql: true`

File: src/js/graphql_endpoint/server.js [hunk 2]
```diff
@@ -56,10 +56,17 @@ const middleware = [cors(corsDelegate), validateJwt];
...
+    }));
+} else {
+    app.use('/graphQlEndpoint/{+proxy}', middleware, graphqlHTTP({
+        schema: schema,
+        graphiql: IS_LOCAL !== null
```
Comment (colin-grapl) [inline src/js/graphql_endpoint/server.js:67]: This can just be `graphiql: false`

**Review by colin-grapl — APPROVED** [review]
Minor comment

File: src/js/graphql_endpoint/server.js (hunk 1 above)
Comment (andrea-grapl) [reply src/js/graphql_endpoint/server.js]: Changed.

File: src/js/graphql_endpoint/server.js (hunk 2 above)
Comment (andrea-grapl) [reply src/js/graphql_endpoint/server.js]: Also changed

**Review by jgrillo-grapl — APPROVED** [review]
LGTM (with Colin's changes)

**Pull request merged**